perfkitbenchmarker/benchmarks/hadoop_terasort_benchmark.py

```diff
@@ -159,33 +94,29 @@ def Run(benchmark_spec):
   vms = benchmark_spec.vms
   master = vms[0]
 
-  hadoop_cmd = ('hadoop-%s/bin/hadoop jar '
-                'hadoop-%s/share/hadoop/mapreduce/'
-                'hadoop-mapreduce-examples-%s.jar ') % (HADOOP_VERSION,
-                                                        HADOOP_VERSION,
-                                                        HADOOP_VERSION)
-  master.RemoteCommand('%s teragen %s /teragen'
-                       % (hadoop_cmd, FLAGS.terasort_num_rows))
-  num_cpus = 0
-  for vm in vms[1:]:
-    num_cpus += vm.num_cpus
+  hadoop_cmd = ('{0} jar {1}/share/hadoop/mapreduce/'
+                'hadoop-mapreduce-examples-{2}.jar').format(
+                    os.path.join(hadoop.HADOOP_BIN, 'yarn'),
```

Comment:
seems inconsistent to use os.path.join here since it's concatenated with a path that uses hardcoded '/'. If a slash separator is ok, can just put /yarn/ into the fixed part of the path. If it's not ok, should use os.path.join consistently for all the path components.